rest-api/tools/import_organizations.py

```diff
@@ -181,13 +181,26 @@ def _entity_from_row(self, row):
                 link=link)
 
   def _get_lat_long_for_site(self, address_1, city, state):
+    full_address = address_1 + ' ' +  city + ' ' + state
     try:
         api_key = os.environ.get('API_KEY')
         gmaps = googlemaps.Client(key=api_key)
         geocode_result = gmaps.geocode(address_1 + '' +  city + ' ' +  state)[0]
-        latitude = geocode_result['geometry']['location']['lat']
-        longitude = geocode_result['geometry']['location']['lng']
-        return latitude, longitude
+        if geocode_result:
+          try:
+            latitude = geocode_result['geometry']['location']['lat']
+            longitude = geocode_result['geometry']['location']['lng']
+            if latitude != None and longitude != None:
```

Comment:
I don't think this check is necessary -- if ['lat'] and ['lng'] don't cause KeyError, it won't be None.

You could instead do:

geometry = geocode_result.get('geometry')
if geometry:
  location = geometry.get('location')
  if location:
    latitude  = geometry.get('lat')
    longitude = geometry.get('lng')
    if latitude and longitude:
     ...

and log warnings in the else blocks.  Then you don't need to catch KeyError below.
 
Have you tested this logic with bad addresses locally?